Sources/TokamakCore/MountedViews/MountedCompositeElement.swift

```diff
@@ -27,28 +27,23 @@ class MountedCompositeElement<R: Renderer>: MountedElement<R>, Hashable {
     hasher.combine(ObjectIdentifier(self))
   }
 
-  var mountedChildren = [MountedElement<R>]()
   let parentTarget: R.TargetType
 
   var state = [Any]()
   var subscriptions = [AnyCancellable]()
-  var environmentValues: EnvironmentValues
```

Comment:
Both `environmentValues` and `mountedChildren` now live in the base `MountedElement` superclass as there are no subclasses that don't need them except the `MountedEmptyView`. I think the overhead of `MountedEmptyView` having empty children and environment containers on it is justified for now when considering the simplification we get here.